webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala

```diff
@@ -391,8 +391,8 @@ class VolumeTracingService @Inject()(
         request.segmentId,
         request.subsamplingStrides,
         request.scale,
-        request.mapping,
-        request.mappingType
```

Comment:
It makes sense, that these are not passed because we are in the VolumeTracingService and not in the EditableMappingService, but the latter doesn't pass these mapping parameters, either. See https://github.com/scalableminds/webknossos/pull/6458/files?diff=split&w=1#diff-5a1cef6ee4682e274fee1f59abbba1bbe211a63e0142a7cea4f61b2a580767bdR596
Where are these parameters passed then? I can't really wrap my head around it.

Reply:
Understandable :D Only in the datastore, where the backend does not know which mapping is active, there is a need to pass this parameter with the request. In the tracingstore, however, the backend knows the selected mapping, and the data is already mapped before it is passed to the isosurface service. This goes for both editable mapping annotations and normal volume annotations.

Reply:
ah, makes sense :)